### PR TITLE
Service Connection

### PR DESCRIPTION
**Pull request by cjkelling**
- Remove FDC_Service
- Connect with the gut_savvy api and use as new service
- Remove ingredient_list method from the food model
- Foods and dishes now save as integers in the session

### CODE REVIEW

**Comment by rhantak** [comment]
The int vs. string in the session foods and dishes was a great catch, good job all.

File: app/services/gut_savvy_service.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  private
+
+  def conn
+    Faraday.new(url: 'https://gut-savvy-service.herokuapp.com') do |faraday|
+      faraday.headers['Content-Type'] = 'application/json'
+      faraday.adapter Faraday.default_adapter
+    end
+  end
+end
```
Comment (johnktravers) [inline app/services/gut_savvy_service.rb]: Looks great! Finally get to consume the microservice API

File: spec/features/users/user_can_edit_and_delete_dishes_in_meal_creation_spec.rb [hunk 2]
```diff
@@ -9,7 +9,9 @@
       @user = User.last
     end
 
-    it "I can remove a dish from the meal I am building" do
+    it 'I can remove a dish from the meal I am building' do
+      Faker::UniqueGenerator.clear # Clears used values for all generators
+      Ingredient.destroy_all
```
Comment (johnktravers) [inline spec/features/users/user_can_edit_and_delete_dishes_in_meal_creation_spec.rb]: Very handy way to generate unique faker data

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -102,7 +102,15 @@
...
-        expect(result.first.keys).to eq([:name, :avg_gut_feeling])
+        expect(result.first.keys).to eq(%i[name avg_gut_feeling])
+      end
+
+      it 'gut_feelings_over_time' do
+        result = @user.gut_feelings_over_time
+
+        expect(result.count).to eq(4)
+        expect(result[0][:avg_gut_feeling]).to eq(-0.4e1)
+        expect(result[0][:date]).to eq(@meal_1.created_at.to_date)
```
Comment (johnktravers) [inline spec/models/user_spec.rb]: Thanks for testing this method. Looks great!

**Pull request merged**